### PR TITLE
fix: only allowed channels

### PR DESCRIPTION
**Pull request by NeonGamerBot-QK**
lol im the only one to abuse bug 
the code uses `respond` so even if its not in the channel it can respond

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -253,8 +253,15 @@ app.view("conduct_report", async ({ ack, view, client }) => {
   }
 });
 
-app.command("/prevreports", async ({ command, ack, client }) => {
+app.command("/prevreports", async ({ command, ack, client, respond }) => {
   await ack();
+  if(!ALLOWED_CHANNELS.includes(command.channel_id)) {
+  respond({
+text: `You are not in the correct channel for this :P`,
```
Comment (SkyfallWasTaken) [inline app.js:260]: Pretty sure you need to `await` respond here, also please fix the formatting :heavysob:

**Comment by v1peridae** [comment]
lyla IS going to be my downfall 😭

File: app.js (hunk 1 above)
Comment (NeonGamerBot-QK) [reply app.js:260]: whoops i did this on github file editor

Comment (NeonGamerBot-QK) [reply app.js:260]: oop well its merged to late :P

**Comment by NeonGamerBot-QK** [comment]
> lyla IS going to be my downfall 😭

time to continue the bug testing :P

**Pull request merged**